pkg/userinfo/roleRef.go

```diff
@@ -166,26 +165,25 @@ func IsRoleAuthorize(rbLister rbaclister.RoleBindingLister, crbLister rbaclister
 				}
 			}
 		}
-	} else {
-		// User or Group
-		excludeDevelopmentRole := []string{"minikube-user", "kubernetes-admin"}
-		for _, e := range excludeDevelopmentRole {
-			if strings.Contains(request.UserInfo.Username, e) {
-				return false, nil
-			}
+	}
```

Comment:
Need to add default return value in `if strings.Contains(request.UserInfo.Username, SaPrefix) {}`, as here it's an if-else condition.